lib/inferno/dsl/fhir_evaluation/rules/all_search_parameters_have_examples.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require_relative '../reference_extractor'
+
+module Inferno
+  module DSL
+    module FHIREvaluation
+      module Rules
+        class AllSearchParametersHaveExamples < Rule
+          attr_accessor :unused_resource_urls
+
+          def check(context)
+            unless ENV.fetch('FHIRPATH_URL')
```

Comment:
`fetch` will raise an error if `FHIRPATH_URL` is not defined.

Reply:
Used ENV[] instead